mslib/mscolab/utils.py

```diff
@@ -30,6 +30,54 @@
 from mslib.mscolab.conf import mscolab_settings
 
 
+def enable_navbar_action_buttons(_type, view_window):
```

Comment:
Not the best object-oriented programmer here, but wouldn't this be better a method of the views itself instead of a function?

Reply:
I agree. The _type can also be determined by the view_window class name through `isinstance()` so both parameters can effectively be removed then.

Reply:
agreed